pkg/gofr/datasource/dgraph/dgraph.go

```diff
@@ -200,18 +191,14 @@ func (d *Client) Mutate(ctx context.Context, mu any) (any, error) {
 		Duration: duration,
 	}
 
-	span.SetAttributes(
-		attribute.String("dgraph.mutation.query", mutationToString(mutation)),
-		attribute.Int64("dgraph.mutation.duration", duration),
-	)
-
 	if err != nil {
 		d.logger.Error("dgraph mutation failed: ", err)
 		ql.PrettyPrint(d.logger)
+
 		return nil, err
 	}
 
-	d.sendOperationStats(ctx, ql, "dgraph_mutate_duration")
+	defer d.sendOperationStats(tracedCtx, start, mutationToString(mutation), "mutate", span, ql, "dgraph_mutate_duration")
```

Comment:
I feel we should remove defer as we are just returning the response and error after this.

Reply:
sure removed defer call to this method, and instead called it directly